run.py

```diff
@@ -443,47 +428,22 @@ def stopdefendwifi(regdwordval):
         accessmask = _winreg.KEY_ALL_ACCESS
 
     # Windows Defender and WifiSense keys
-<<<<<<< HEAD
     wdwfsdict = 
-    { 'Windows Defender Delivery Optimization Download': [HKEY_LOCAL_MACHINE, r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config', 'DODownloadMode', regdwordval],
-      'WifiSense Credential Share': [HKEY_LOCAL_MACHINE, r'SOFTWARE\Microsoft\WcmSvc\wifinetworkmanager\features', 'WiFiSenseCredShared', regdwordval],
-      'WifiSense Open-ness': [HKEY_LOCAL_MACHINE, r'SOFTWARE\Microsoft\WcmSvc\wifinetworkmanager\features', 'WiFiSenseOpen', regdwordval],
-      'Windows Defender Spynet': [HKEY_LOCAL_MACHINE, r'SOFTWARE\Microsoft\Windows Defender\Spynet', 'SpyNetReporting', regdwordval],
-      'Windows Defender Sample Submission': [HKEY_LOCAL_MACHINE, r'SOFTWARE\Microsoft\Windows Defender\Spynet', 'SubmitSamplesConsent', regdwordval]}
+    { 'Windows Defender Delivery Optimization Download': [HKEY_LOCAL_MACHINE, r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config', 'DODownloadMode', REG_DWORD, regdwordval],
```

Comment:
Having HKEY_LOCAL_MACHINE in these lists doesn't work, as it's not defined. You have to use _winreg.HKEY_LOCAL_MACHINE